src/Leaflet.VectorGrid.js

```diff
@@ -217,6 +217,8 @@ L.VectorGrid = L.GridLayer.extend({
 		switch (feat.type) {
 		case 1:
 			layer = new PointSymbolizer(feat, pxPerExtent);
+			//prevent leaflet from treating these canvas points as real markers
+			layer.getLatLng = null;
```

Comment:
I think this is a fine solution, at least for now, given how LVG hacks a lot of other internals of Leaflet. :+1: